templates/single-page.twig

```diff
@@ -7,13 +7,13 @@
 
 	<div class="page-template">
 
-		<form id="password-form" class="" action="{{login_url}}?action=postpass" method="post">
+		<form id="password-form" class="password-form" action="{{login_url}}?action=postpass" method="post">
 			<div class="mb-3">
 				<label for="pwbox-{{post.ID}}" class="form-label"><h6>{{ __( 'To see the content of this page, please enter your password  below', 'planet4-master-theme' ) }}</h6></label>
-				<div class="col-sm-8">
-					<input class="form-control password-field" name="post_password" id="pwbox-{{post.ID}}" type="password">
+				<div class="input-text">
+					<input name="post_password" id="pwbox-{{post.ID}}" type="password" {% if not is_password_valid %}class="is-invalid"{% endif %}>
+					{% if not is_password_valid %} <div class="invalid-pw-feedback">{{ __( 'This field is required', 'planet4-master-theme' ) }}</div>{% endif %}
```

Comment:
I find this message a bit confusing, shouldn't it say something more along the lines of "Invalid password, please try again"? 🤔 Unless the user actually didn't enter anything

Reply:
Yes, we can apply this message @mleray and @sagarsdeshmukh